mautic/mautic
# Add support for upcoming webinars and  add start date to event name for Citrix services

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | staging for features or enhancements / 3.0 for bug fixes 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:

This PR added start date of event after event name in choosen in campaign and segments. Some of webinars with same name require this as unique element.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Setup Citrix plugin and Go To Webinar https://docs.mautic.org/en/plugins/citrix
3. Sync  data `php bin/console mautic:citrix:sync`
4. Go to segments, go to campaigns and check Go To Webinar condition and action
5.  See selectbox  event names with start date suffix

![image](https://user-images.githubusercontent.com/462477/94438480-74722e80-019f-11eb-816d-7f89a25c7c23.png)
6. Create upcoming webinar
7. Go to Mautic and check segment filter : webinar attended and campaign condition for webinar attended. See the webinar is not in the list. After PR you should see it.





<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/9252"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
